apps/civil/civil-general-applications/demo.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   values:
     java:
       ingressClass: traefik-private
-      image: hmctspublic.azurecr.io/civil/general-applications:prod-8906f06-20230328142412 #{"$imagepolicy": "flux-system:civil-general-applications"}
+      disableIngressClassAnnotation: true
+      image: hmctspublic.azurecr.io/civil/general-applications:pr-951-dac8b29-20230328163110 #{"$imagepolicy": "flux-system:demo-civil-general-applications"}
```

Comment:
update with latest hash value . . or just leave it as pr-951 as bot would do auto commit with latest image  ..this is required as both ga and civil service needs to have latest java 17